boilerplate/app/models/helpers/setupRootStore.ts

```diff
@@ -31,8 +31,8 @@ export async function setupRootStore(rootStore: RootStore) {
     applySnapshot(rootStore, restoredState)
   } catch (e) {
     // if there's any problems loading, then inform the dev what happened
-    if (__DEV__) {
-      console.tron.error(e.message, null)
+    if (__DEV__ && e instanceof Error) {
+      console.tron.error?.(e.message, null)
     }
```

Comment:
If you format it this way, does this get stripped in the production build?

From what I understand it's a little finicky and we might have to do something like this:

```suggestion
    if (__DEV__) {
      if (e instanceof Error) console.tron.error?.(e.message, null)
    }
```

Reply:
Unsure but I'm fine to change it!